deployment/hasura/metadata/databases/AerieMerlin/tables/public_activity_directive_changelog.yaml

```diff
@@ -0,0 +1,81 @@
+table:
+  name: activity_directive_changelog
+  schema: public
+object_relationships:
+- name: activity_directive
+  using:
+    manual_configuration:
+      remote_table:
+        name: activity_directive
+        schema: public
+      column_mapping:
+        plan_id: plan_id
+        activity_directive_id: id
+select_permissions:
+  - role: aerie_admin
+    permission:
+      columns:
```

Comment:
Minor, nonblocking: This formatting is inconsistent with our other `.yaml`s

Reply:
I figured this formatting more closely mirrors the `sql` schema definitions, and is easier to view diffs of.